OneSignalSDK/onesignal/src/main/java/com/onesignal/ADMMessageHandlerJob.kt

```diff
@@ -0,0 +1,46 @@
+package com.onesignal
+
+
+import android.content.Context
+import android.content.Intent
+import com.amazon.device.messaging.ADMMessageHandlerJobBase
+
+class ADMMessageHandlerJob : ADMMessageHandlerJobBase() {
```

Comment:
Add this class to the `consumer-proguard-rules.pro`. See comment in this file
https://github.com/OneSignal/OneSignal-Android-SDK/blob/master/OneSignalSDK/onesignal/consumer-proguard-rules.pro#L56